guidelines/responses.md

```diff
@@ -97,25 +97,27 @@ In case of error, the response object will extend the simple response object wit
+| `rateCode` | `string`  | optional\* | Rate code which caused the error. This is _required_ for rate errors and rate category errors. The rate will be automatically unsynchronized in Mews. |
 
+> **\*Important**: If the error is a category error (code 10) or rate category error (code 11), then `categoryCode` must be specified.
+> If the error is a rate error (code 9) or rate category error (code 11), then `rateCode` must be specified.
 
 ### Error codes
 
 | Code | Description |
 | :-- | :-- |
 | `1` | **System error**<br>Unspecified system error. Message may be re-sent after an interval of time. |
-| `2` | **Reservation error**<br>Reservation does not exist. When this error code is returned, it should be acknowledged and the reservation should _not_ be re-tried. |
-| `3` | **Property error**<br>Property does not exist. |
-| ~~`4`~~ | ~~**Space error**<br>Space type does not exist.~~ (deprecated - use code 10 instead) |
-| ~~`5`~~ | ~~**Rate error**<br>Rate plan does not exist.~~ (deprecated - use code 9 instead) |
-| `6` | **Validation error**<br>Validation error, e.g. invalid value "XXX" of field "YYY". |
-| `7` | **Processing error**<br>Processing error, e.g. processing of the booking would violate some internal PMS limitation. |
-| `8` | **Invalid authorization** |
-| `9` | **Rate error**<br>Rate error, e.g. rate plan XX prices updates cannot be accepted due to the settings in the channel manager extranet. The affected rate code must be supplied in the `rateCode` property. This rate code will then be unsynchronized. |
-| `10` | **Category error**<br>Category error, e.g. space category XX availability updates cannot be accepted due to the settings in the channel manager extranet. The affected category code must be supplied in the `categoryCode` property. The category will then be deleted from the rate. | 
-| `11` | **Rate category error**<br>Rate category error, e.g. category was removed from the rate plan in channel manager extranet. The affected rate code and category code must be supplied. The category will be deleted from the rate. |
-| `12` | **Availability error**<br>Availability configuration error, e.g. availability updates blocked from the channel manager extranet. This error automatically disables availability updates. |
-| `13` | **Prices error**<br>Prices configuration error, e.g. price updates blocked from the channel manager extranet. This error automatically disables price updates. |
-| `14` | **Restrictions error**<br>Restrictions configuration error, e.g. restriction updates blocked from the channel manager extranet. This error automatically disables restrictions updates. |
\ No newline at end of file
+| `2` | **Reservation not found**<br>The specified reservation could not be found in the system. When this error code is returned, it should be acknowledged and the reservation should _not_ be re-tried. |
+| `3` | **Property not found**<br>The specified property, hotel or other enterprise could not be found in the system. |
```

Comment:
this is technically connection (i.e. integration), not property (hotel, enterprise).

so maybe
```suggestion
| `3` | **Connection not found**<br>The specified connection (integration) could not be found in the system. |
```